index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
With regard to the returned entries from `window.performance.getEntries()` and those emitted to the PO callback when `buffered: true`:

@igrigorik are we OK with [breaking reverse-compatibility for RT/UT](https://github.com/w3c/performance-timeline/pull/76#discussion_r109477186)? Or is having a discrepancy between RT/UT and ST more preferred?